CGPs/cgp-0194.md

```diff
@@ -128,12 +133,12 @@ Experienced Grants and Public Goods Program Developer. Head of Grants at [Gitcoi
 
 ## Proposed Changes
 ### Transactions
-Unique Transaction of ```six hundred eighty five thousand (685,000)``` Celo tokens approval to Multisign ```0x85910bE70D7eBF149918ed96eF8BE175A0639c33```
+Unique Transaction of ```six hundred eighty five thousand (875,000)``` Celo tokens approval to Multisign ```0x85910bE70D7eBF149918ed96eF8BE175A0639c33```
```

Comment:
This still has the wrong written numerical values. Same as the other line change